### ISSUE TITLE
401 Response when run.

### ISSUE THREAD

**Issue by alexandertomsovic** [elebumm/RedditVideoMakerBot#862]
**Describe the bug**
When I type "python main.py" I encounter a message saying "Using subreddit: r/AskReddit from environment variable config" followed by a 401 response which kills the program. Any fixes?

**Comment by William9923** [comment]
No, 401 (Unauthorized) happens if you enter wrong credential in the .env files. Kindly double check your credential for the bot 🙏
